packages/components/.storybook/buttonUtils.stories.module.css

```diff
@@ -0,0 +1,24 @@
+.recommendedVariantSet,
+.recommendedVariantSet h2,
+.recommendedVariantSet button,
+.recommendedVariantSet a {
+  @apply mb-4;
```

Comment:
nit: is it easier to put this in a `grid`? was thinking it might be nice to fill the horizontal space more anyways

Reply:
>was thinking it might be nice to fill the horizontal space more anyways

Do you mean you'd like to see them laid out in rows so they take up less vertical space?

Reply:
yeah, but now that they're separate stories I don't mind it